Sources/Core/Network/Client+Async Request.swift

```diff
@@ -0,0 +1,137 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import Alamofire
+import CocoaLumberjack
+
+@available(iOS 13, *)
+public extension Client {
+	/// Shortcut method for building the request and loading the response (ignoring data).
+	///
+	/// - parameter request: The router request type
+	/// - parameter automaticallyCancelling: Bool determining whether or not the request should be cancelled when the enclosing async context is cancelled.
+	///
+	/// - returns: the result
+	func requestVoid(
+		_ request: RouterType,
+		automaticallyCancelling: Bool = false
+	) async -> Result<Void, Error> {
+		let task = self.request(request).serializingResponse(using: PassthroughResponseSerializer(), automaticallyCancelling: automaticallyCancelling)
```

Comment:
Seems weird we'd need this? The non-async versions (i.e. with callbacks) don't need a custom serializer.
- If this is "the new way" of doing things, shouldn't it also be applied to the old versions? 
- You only use it here, not in the other `request…` versions